src/Pack1/MyClass.java

```diff
@@ -0,0 +1,15 @@
+package Pack1;
+
+class MyClass {
+
+    public static int myMethod(int a, int b) {
+        return a + b;
+    }
+
+    public static void main(String[] args) {
+        int a = 100;
+        int b = 200;
+        int c = myMethod(a, b);
```

Comment:
it's ok but for code optimization it's a good idea to write once int c = myMethod(100, 200);